gitlab/v4/objects/projects.py

```diff
@@ -3,6 +3,8 @@
 https://docs.gitlab.com/ee/api/projects.html
 """
 
+from io import BufferedReader
```

Comment:
Please do

```
import io
```

And then use `io.BufferedReader`

Reply:
Done :+1: